src/number-formatting/format.js

```diff
@@ -105,10 +107,12 @@ function formatNumber(inputNumber, decimals, options) {
     absoluteNumber = Math.round(absoluteNumber * factor) / factor;
 
     let formattedNumber = expandNumber(Math.abs(absoluteNumber),
-                                decimals,
-                                options.groupSizes,
-                                options.groupSeparator,
-                                options.decimalSeparator);
+                                decimals, {

```

Comment:
for readability I would move payload to separate const and use it as expandNumber argument.

```expandNumber(Math.abs(absoluteNumber, decimals, formatOptions);```